ajax/libs/big-integer/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "big-integer",
+  "filename": "BigInteger.min.js",
+  "version": "1.6.26",
+  "description": "An arbitrary length integer library for Javascript",
+  "keywords": [
+    "math",
+    "big",
+    "bignum",
+    "bigint",
+    "biginteger",
+    "integer",
+    "arbitrary",
+    "precision",
+    "arithmetic"
+  ],
+  "author": "Peter Olson <peter.e.c.olson+npm@gmail.com>",
+  "license": "Unlicense",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/peterolson/BigInteger.js.git"
+  },
+  "npmName": "big-integer",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "*.min.js"
```

Comment:
cdnjs need pre-minified file `BigInteger.js`
Let's use more flexible pattern to only exclude `BigInteger.d.ts`(not for front-end) & `BigInteger.test.js`(just for test).

Reply:
I have modified the pattern to be `BigInteger!(*.test.*|*.d.*)`.
Let me know if it's feasible, thanks.

Reply:
Nice @cy91244 
Just one point: `*.d.ts` could be better than `*.d.*` because typescript format postfix.

Reply:
@extend1994 
Aw, you are right!
I have modified the commit, please confirm, thank you.